common/api/core-frontend.api.md

```diff
@@ -8830,6 +8839,12 @@ export interface RenderPlan {
     // (undocumented)
     readonly clipStyle: ClipStyle;
     // (undocumented)
+    readonly ellipsoidCenter?: Point3d;

```

Comment:
Replace `isGlobeModeEllipsoid` and the three new ellipsoid properties with an optional object property containing those three properties called `ellipsoid`, present if and only if globe mode is ellipsoid.
If the globe is an ellipsoid we need all three of those properties to be defined.

Reply:
Does the type of `ellipsoid` need to be any more substantial than an inline type that contains the 3 other properties?

All the other types on `RenderPlan` are classes, and wasn't sure if something similar should be made here.